src/lib/core/common-behaviors/color.ts

```diff
@@ -24,8 +24,8 @@ export interface HasRenderer {
 export type ThemePalette = 'primary' | 'accent' | 'warn' | null;
 
 /** Mixin to augment a directive with a `color` property. */
-export function mixinColor<T extends Constructor<HasRenderer>>(base: T, defaultColor?: ThemePalette)
-    : Constructor<CanColor> & T {
+export function mixinColor<T extends
+    Constructor<HasRenderer>>(base: T, defaultColor?: ThemePalette): Constructor<CanColor> & T {
```

Comment:
It would be better to break after the `(`

As @DevVersion says, though, this is passing on CI. Do you have a stale version of tslint installed?